doc/source/whatsnew/v0.19.0.txt

```diff
@@ -229,6 +229,7 @@ Other enhancements
 - ``pd.read_html()`` has gained support for the ``decimal`` option (:issue:`12907`)
 - A top-level function :func:`union_categorical` has been added for combining categoricals, see :ref:`Unioning Categoricals<categorical.union>` (:issue:`13361`)
 - ``Series`` has gained the properties ``.is_monotonic``, ``.is_monotonic_increasing``, ``.is_monotonic_decreasing``, similar to ``Index`` (:issue:`13336`)
+- ``DataFrame.sort_values()`` has gained support for re-ordering columns by index label (:issue:`10806`)
```

Comment:
The rewording in that commit is good! (exactly what I wanted to suggest to combine both :-))

Maybe just "with" after "columns"